opentimelineio/opentime.py

```diff
@@ -233,9 +234,9 @@ class TimeTransform(object):
     """1D Transform for RationalTime.  Has offset and scale."""
 
     def __init__(self, offset=RationalTime(), scale=1.0, rate=None):
-        self.offset = offset
-        self.scale = scale
-        self.rate = rate
+        self.offset = copy.copy(offset)
```

Comment:
You don't need to copy primitives ? i.e. int , float etc.

Reply:
Offset is a RationalTime.  Scale and Rate are primitives though, I'll fix that.